### DISCUSSION TITLE
[Ollama] Local modes can not be loaded

### DISCUSSION THREAD

**Discussion by rhiisi** [darrenburns/elia#62] [Questions]
Hello, I just find this fantastic TUI. But it seems like the Ollama models can not be loaded?
<img width="435" alt="image" src="https://github.com/darrenburns/elia/assets/10807209/528f1253-3f29-491f-94f9-e0fd4604491e">

This is what I added to the  config.toml & Ollama is running on the server mode.

But I still cant load the model to Edia.

<img width="786" alt="image" src="https://github.com/darrenburns/elia/assets/10807209/313ff303-af91-477c-b727-d7edde4344b6">

Anything wrong the my configuration maybe?

**Comment by darrenburns** [comment]
I checked and it still seems to work with local models for me. Your config looks correct. I tested it with ollama/llama3. Unfortunately my machine can't run the model you're trying so I can't check that.
